project_wsjf/models/project_task.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2019  Gabriel Cardoso de Faria - KMEE - www.kmee.com.br
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+from odoo import models, fields, api
+
+
+class ProjectTask(models.Model):
+    _name = 'project.task'
+    _inherit = [
+        'project.task',
+        'project.wsjf.mixin',
+    ]
+    _order = 'wsjf desc, sequence'
+
+    project_id = fields.Many2one(
+        inverse='_inverse_project_id',
```

Comment:
Can you explain this?
AFAIK project_id is not a computed field, why define an Inverse computation function for it?